tests/diagnostics/uninitialized-out-parameters.slang

```diff
@@ -47,5 +47,48 @@ void control_flow_defined(bool b, out float y)
     return;
 }
 
+// Specialized handling for intrinsic asm
+void instrincs_defined(float x, out float exp)
+{
+    __intrinsic_asm "frexp";
+}
```

Comment:
FYI, the following case should print a warning but I assume that it will be not easy to implement.
```
void instrincs_defined(float x, out float exp)
{
    __intrinsic_asm "frexp(x)";
}
```
We will need to ignore this case for now.